simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/HTTPServerBasicAuthenticator.java

```diff
@@ -0,0 +1,38 @@
+package io.prometheus.client.exporter;
+
+/**
+ * Basic HTTP authenticator
+ */
+public class HTTPServerBasicAuthenticator extends com.sun.net.httpserver.BasicAuthenticator {
+
+    private static final String REALM = "/";
+
+    private String username;
+    private String password;
```

Comment:
Nitpick: You could make these final to make it clear that `HTTPServerBasicAuthenticator` is an immutable class.

Reply:
Changed.